Bottom: e19f8b25ba4b07db559d96b7fe7bb3810ef0fc58
Top:    4aedeff3fbdc7edda79bd9217b52850a061a12bf
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 20:19:13 +0000

Remove debug from outline generation


---

diff --git a/src/polygon.c b/src/polygon.c
index 5e4e1fc..2ba4f52 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -108,7 +108,7 @@ dicer output is used for HIDs which cannot render things with holes
 #include <sys/time.h>
 #include <sys/resource.h>
 
-#define DEBUG_CIRCSEGS
+#undef DEBUG_CIRCSEGS
 
 #define ROUND(x) ((long)(((x) >= 0 ? (x) + 0.5  : (x) - 0.5)))
 
diff --git a/src/polygon1.c b/src/polygon1.c
index f34c968..c323758 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -115,7 +115,7 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 //#define DEBUG_JUMP
 //#define DEBUG_GATHER
 #undef DEBUG_ANGLE
-//#define DEBUG
+#undef DEBUG
 #ifdef DEBUG
 #define DEBUGP(...) pcb_fprintf(stderr, ## __VA_ARGS__)
 #else
